### PR TITLE
Fix chart margins and add ability to add suffixes to axis

### PR DESCRIPTION
**Pull request by deekerno**
## Summary
Enforces consistent margins on each of the charts. Also adds ability to add suffixes to numbers on left axis.

## Testing Plan
Tested manually.

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/components/Charts/GeneralChart.tsx [hunk 1]
```diff
@@ -12,6 +12,8 @@ import {
 
 import Metric from '../../types/Metric'
 import GeneralChartBackground from './GeneralChartBackground'
+import { TickFormatter } from '@visx/axis'
+import { AnyD3Scale, ScaleInput } from '@visx/scale'
```
Comment (dguenther) [inline src/components/Charts/GeneralChart.tsx:16]: I know we don't have sort turned on, but can these imports go up by the other `@visx` imports?

File: tsconfig.json [hunk 2]
```diff
@@ -14,8 +18,12 @@
...
+    "jsx": "react-jsx"
   },
-  "include": ["src"],
-  "exclude": ["src/assets/images/*"]
+  "include": [
+    "src"
+  ],
+  "exclude": [
+    "src/assets/images/*"
+  ]
```
Comment (dguenther) [inline tsconfig.json:28]: Are changes to tsconfig/yarn.lock related, or can we put them in a separate PR?

File: src/assets/jss/theme.tsx [hunk 3]
```diff
@@ -47,7 +47,7 @@ declare module '@material-ui/core/styles/createPalette' {
   }
 }
 
-const theme = createMuiTheme({
+const theme = createTheme({
```
Comment (dguenther) [inline src/assets/jss/theme.tsx:50]: Is this related?

**Review by dguenther — CHANGES_REQUESTED** [review]

File: src/components/Charts/GeneralChart.tsx (hunk 1 above)
Comment (deekerno) [reply src/components/Charts/GeneralChart.tsx:16]: Sure, I'll move them.

**Review by dguenther — APPROVED** [review]

**Review by dguenther — APPROVED** [review]
